Tryouts/Prototypes/Shell/App.xaml.cs

```diff
@@ -28,6 +28,11 @@ namespace Shell
     /// </summary>
     public partial class App : Application
     {
-        
+        public string[] CustomerCode { get; set; }
```

Comment:
What customer code? Who's the customer and what kind of code is provided here? It's actually populated from command line arguments, so why isn't it called `CommandLineArguments`?